pkg/scheduler/service/service.go

```diff
@@ -547,7 +547,8 @@ func (s *SchedulerServer) bootstrap(hks hyperkube.Interface) (*ha.SchedulerProce
 		ReconcileCooldown: s.ReconcileCooldown,
 	})
 
-	masterUri := kmcloud.MasterURI()
+	kmcloud.AwaitInitialization() // Blocking
```

Comment:
do we need to invoke cloudprovider.InitCloudProvider(...) before waiting? Not sure that this blocking call will ever return unless we do.